ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalGoverningBody.cshtml.cs

```diff
@@ -34,6 +34,12 @@ public async Task<IActionResult> OnPostAsync(int id)
 
 			await LegalRequirementsRepository.UpdateByProjectId(id, LegalRequirements);
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
+			if (ReturnPage(returnPage))
```

Comment:
Could move this into the LegalModelBase.cs

Reply:
True!